fhir-audit/src/main/java/com/ibm/fhir/audit/logging/api/impl/NopService.java

```diff
@@ -1,38 +1,42 @@
 /*
- * (C) Copyright IBM Corp. 2016,2019
+ * (C) Copyright IBM Corp. 2016,2020
  *
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package com.ibm.fhir.audit.logging.impl;
+package com.ibm.fhir.audit.logging.api.impl;
```

Comment:
Why the `.api` in the package name?  Can we just use `com.ibm.fhir.audit.logging.impl` for the impl classes?

Reply:
legacy -- I can change it... matched the existing package structure

Reply:
I pinged paul on the side and suggested that maybe we could even do away with the `logging` in the package name for all classes in this module.  My understanding is there is no other audit package other than logging and so this level seems totally pointless.

Reply:
Changed per discussion

Reply:
Discussed in chat